### PR TITLE
ZK-4653: Provide a collection of constants for Keys and provide isPre…

### PR DESCRIPTION
**Pull request by plijyfes**
…ssed API

### CODE REVIEW

File: zk/src/archive/web/js/zk/evt.js [hunk 1]
```diff
@@ -208,6 +208,45 @@ evt.stop({progagation:true,revoke:true}); //revoke the event propagation
...
+	 * @see zKeys
+	 */
+	isPressed: function () {
+		var keyCount = 0;
+		for (var i = 0, len = arguments.length; i < len; i++) {
+			var arg = arguments[i];
+			if (arg != zKeys.META && arg != zKeys.ALT && arg != zKeys.CONTROL && arg != zKeys.SHIFT) {
+				keyCount++;
+				if (keyCount > 1)
+					return false; // Keys combination is invalid
```
Comment (plijyfes) [inline zk/src/archive/web/js/zk/evt.js:229]: zk.error

File: zk/src/archive/web/js/zk/evt.js [hunk 2]
```diff
@@ -208,6 +208,45 @@ evt.stop({progagation:true,revoke:true}); //revoke the event propagation
...
+				else if (arg != this.key)
+					return false;
+			} else {
+				switch (arg) {
```
Comment (plijyfes) [inline zk/src/archive/web/js/zk/evt.js:233]: if

File: zk/src/archive/web/js/zk/keys.js [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+		
+		History:
+				Tue Sep 01 11:59:55 CST 2020, Created by leon
+
+Copyright (C) 2020 Potix Corporation. All Rights Reserved.
+*/
+
+/**
+ * A collection of constants for KeyboardEvent.key values.
+ * Ref: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values
```
Comment (plijyfes) [inline zk/src/archive/web/js/zk/keys.js:15]: Since 9.5.0

**Review by scribetw — CHANGES_REQUESTED** [review]
1. keys.js Javadoc
2. need some refine

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**